packages/backend/src/config/Config.ts

```diff
@@ -79,6 +79,7 @@ export interface ActivityV2Config {
   readonly starkexApiKey: string
   readonly starkexApiDelayHours: number
   readonly starkexCallsPerMinute: number
+  readonly starkexResyncLastDays: number
```

Comment:
I don't see a reason why this would change depending on the environment. That being said I think it will be cleaner and simpler to add this to `StarkexLayer2TransactionApiV2` and define per project (even if at this point all of them will have the same value).